specification/cognitiveservices/data-plane/Language/stable/2022-05-01/textanalytics.json

```diff
@@ -0,0 +1,2694 @@
+{
```

Comment:
Based on the [diff](https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL3Jhdy5naXRodWJ1c2VyY29udGVudC5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvZGV2LWNvZ25pdGl2ZXNlcnZpY2VzLUxhbmd1YWdlLTIwMjItMDMtMDEtcHJldmlldy9zcGVjaWZpY2F0aW9uL2NvZ25pdGl2ZXNlcnZpY2VzL2RhdGEtcGxhbmUvTGFuZ3VhZ2UvcHJldmlldy8yMDIyLTAzLTAxLXByZXZpZXcvdGV4dGFuYWx5dGljcy5qc29uIiwidGFyZ2V0VXJsIjoiaHR0cHM6Ly9yYXcuZ2l0aHVidXNlcmNvbnRlbnQuY29tL0F6dXJlL2F6dXJlLXJlc3QtYXBpLXNwZWNzLzhmMzIyMzNhOTA1MmRiNjg3NGY0ZDdkNjRiMDU3YTQwOGJkNTQwNDEvc3BlY2lmaWNhdGlvbi9jb2duaXRpdmVzZXJ2aWNlcy9kYXRhLXBsYW5lL0xhbmd1YWdlL3N0YWJsZS8yMDIyLTA1LTAxL3RleHRhbmFseXRpY3MuanNvbiJ9) between 2022-03-01-preview (in the dev-cognitiveservices-Language-2022-03-01-preview branch) and 2022-05-01, there are quite a few breaking changes. I've merely listed them here since I was basing this off the diff view:

- Missing: AnalyzeTextLROTaskKind.ExtractiveSummarization
- Missing: AnalyzeTextLROResultsKind.ExtractiveSummarizationLROResults
- Changed: CustomSingleLabelClassificationResult -> CustomLabelClassificationResult
- Changed: CustomSingleLabelClassificationResult allOf #/definitions/SingleClassificationDocumentResult -> CustomLabelClassificationResult allOf #/definitions/ClassificationDocumentResult
- Missing: SingleClassificationDocumentResult
- Missing: CustomMultiLabelClassificationResult
- Changed: MultiClassificationDocumentResult -> ClassificationDocumentResult
- Missing: ExtractiveSummarizationTaskParameters
- Missing: ExtractiveSummarizationLROTask
- Missing: ExtractiveSummarizationResult
- Missing: ExtractiveSummarizationSortingCriteria
- Missing: ExtractedSummaryDocumentResult
- Missing: ExtractedSummarySentence
- Changed: CustomSingleLabelClassificationLROResult allOf #/definitions/CustomSingleLabelClassificationResult -> CustomSingleLabelClassificationLROResult allOf #/definitions/CustomLabelClassificationResult
- Changed: CustomMultiLabelClassificationLROResult allOf #/definitions/CustomMultiLabelClassificationResult -> CustomMultiLabelClassificationLROResult allOf #/definitions/CustomLabelClassificationResult
- Missing: ExtractiveSummarizationLROResult